### DISCUSSION TITLE
Removal of Unused Data

### DISCUSSION THREAD

**Discussion by rastacalavera** [JupiterBroadcasting/jupiterbroadcasting.com#441] [Ideas]
Hello!
I am trying to find ways to get involved with the project and am thinking this may be a good place to start. I spun up the JB site on my local machine and started making tweaks to figure out how things work. I think that the folder called `hosts` is not being used and probably isn't needed in the project. 

It seems to me that all hosts and guests are located in the `people` folder and are distinguished by tag. Maybe the `hosts` folder is a hold over from the `Castanet` theme? 

Is this needed? Would this be a good first PR for me to do?

**Comment by CGBassPlayer** [comment]
Hey @rastacalavera, this would be a great start! #318 has some good information regarding cleaning up the repo as well. The GItHub action part mentioned there has already been dealt with so that can be ignored.

As far as the hosts folder, it looks like it is being used somehow (though I am not sure how) because when I remove that file, the dev server fails with:

```
failed to render pages: render of "home" failed: "/app/themes/jb/layouts/index.html:28:34": execute of template failed at <$all_most_recent.ByDate.Reverse>: can’t evaluate field ByDate in type []interface {} 
```

But going through and seeing what isn't needed would be a fantastic help and this would something that would be a periodic recurring task to make sure the repo stays lean. But I say start with what we have documented and anything else you come across!
